src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+        Personal information
+        </legend>
+        <div>
+          <label for="Surname">
+            Surname:
+          </label>
+          <input id="Surname" type="text"
+          name="Surname"
+          autocomplete="off"
+          required
+          >
+        </div>
+        <div>
+          <label for="Name">
+            Name:
+          </label>
+          <input type="text" name="Name"
+          id="Name"
+          autocomplete="off"
+          >
```

Comment:
Check the [code style guide](https://mate-academy.github.io/style-guides/htmlcss.html) please
```suggestion
          <input 
            type="text" 
            name="Name"
            id="Name"
            autocomplete="off"
          >
```

Reply:
make change